UI for Forgot Password Page/styles.css

```diff
@@ -0,0 +1,90 @@
+body, html {
+    height: 100%;
+    margin: 0;
```

Comment:
Rename the file to ForgotPasswordPage.module.css. Don't modify the body/html tags for your components.